apps/admin/traefik-no-proxy/kustomization.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+nameSuffix: -no-proxy
 resources:
+  - ../traefik2/traefik2.yaml
   - traefik-no-proxy.yaml
```

Comment:
Can this be moved under `patchesStrategicMerge`? Causes two separate helm releases currently